packages/grafana-schema/src/veneer/dashboard.types.ts

```diff
@@ -49,3 +49,9 @@ export const defaultVariableModel = {
 export const defaultPanel: Partial<Panel> = raw.defaultPanel;
 export const defaultFieldConfig: Partial<FieldConfig> = raw.defaultFieldConfig;
 export const defaultFieldConfigSource: Partial<FieldConfigSource> = raw.defaultFieldConfigSource;
+
+export interface MatcherConfig<TConfig = any> extends raw.MatcherConfig {
+  options?: TConfig;
+}
+
+export const defaultMatcherConfig: Partial<MatcherConfig> = raw.defaultMatcherConfig;
```

Comment:
This seems like a good candidate to automate generation for.

Reply:
Certainly worth considering. Defaults need a fairly significant overhaul due to this fundamental limitation grafana/cuetsy#32, so we should think about any such approach in the context of addressing that problem

That's doubly important because of an idea @joshhunt and i have been circling around - rather than having just one set of defaults that are declared with the schema itself, it may make sense to have multiple, named default sets be part of the overall schema declaration story. That's something i can imagine natively including in thema.